### PR TITLE
Week 4 Update

### PR DESCRIPTION
**Pull request by joshukraine**

### CODE REVIEW

File: source/blog/articles/2019-07-the-last-stretch.html.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+READMORE
+
+Wednesday we went to Lutsk to pass out invites for *Bible First*, the correspondence course. However, when Joshua started the van in the morning, he realized that the dash lights, blinkers, and brake lights did not work. It looks like the van was in a bad mood during this CMO! Our mechanics (who fixed the van during week 2) had a look at the van but could not figure out the problem. So we hopped in Tolik's and Serhii's cars to get to Lutsk. Once there, we distributed tracts in P.O. boxes of apartment blocks, and got out a very good amount of tracts.
+
+The second day of our **literature blitz**, as we call them, also went very well. Cornelius and I did have a lady run us off, but that is not uncommon when we do literature distribution. We just go to the next entrance out of sight and hope nobody will see us as threats! The total amount of *Bible First* invitations we distributed during the two days in Lutsk was 33,000.
```
Comment (ralphukraine) [inline source/blog/articles/2019-07-the-last-stretch.html.markdown]: I would make "literature blitz" italic instead.

File: source/blog/articles/2019-07-the-last-stretch.html.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+We were able to go to church on Sunday, and enjoyed nice Ukrainian hymns played by the church musicians. We then drove to Yasenytsya-Zamkova, where our last film showing was. People started showing up as early as thirty minutes in advance, which is rather uncommon. By 17:00, when the movie started, we already had a significant crowd, peaking to twenty-three during the movie. Just as the movie began playing, we had a good rain outside, meaning the attendants were not in a hurry to leave the hall! The crowd was quite attentive, and even clapped after Josh did his Gospel presentation and outro message.
+
+Once Josh had finished speaking, a man from the audience stood up and introduced himself as a regional government official. He was very complimentary of our presentation, and thanked us for our work. He then introduced a colleague of his who is running for parliament. This man was also very positive about the film showing. He spoke for several minutes about his campaign for parliament, emphasizing the importance of a strong family unit. At the end, we prayed for both men, noting the passage in Scripture which instructs us to pray for those in positions of authority.
```
Comment (ralphukraine) [inline source/blog/articles/2019-07-the-last-stretch.html.markdown]: Perfect! You're the one that understood was what said in that conversation, so thanks for telling the story!

File: source/blog/articles/2019-07-the-last-stretch.html.markdown [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+On Monday morning the guys started packing their stuff, and in the afternoon we did our very last street preaching. Everything went very well, and we had great weather! Cornelius preached first, then Kade, and finally Nelson. The messages were great, and we had a several people fill out Good and Evil invites right there after the messages.
+
+Then the Steele's had us over to their home to have the last team dinner. We had wonderful food, a funny game, and a great time overall. Thanks to the ladies for making all of that possible!
+
+This concludes Carpathian Mountain Outreach 2019. This CMO was simply amazing. The team I got to work with are some of the best folks I have met. God's hand in all of it was very clear. We had our share of obstacles, like rain, mechanical problems, or illness, but the Lord had a plan in everything that happened, and it truly worked out great and to His glory.
+
+I will quote this from Cornelius: **"Don't be sad that it's over, be glad that it happened."** Very true.
```
Comment (ralphukraine) [inline source/blog/articles/2019-07-the-last-stretch.html.markdown]: The quote in italics as well.

File: source/blog/articles/2019-07-the-last-stretch.html.markdown (hunk 1 above)
Comment (joshukraine) [reply source/blog/articles/2019-07-the-last-stretch.html.markdown]: Done

File: source/blog/articles/2019-07-the-last-stretch.html.markdown (hunk 3 above)
Comment (joshukraine) [reply source/blog/articles/2019-07-the-last-stretch.html.markdown]: Done.

**Pull request merged**